src/utils/getRandomImagePlaceholder.ts

```diff
@@ -0,0 +1,12 @@
+import { StaticImageData } from 'next/image';
+
+import PlaceholderOneImage from '@assets/images/placeholders/placeholder1.png';
+import PlaceholderTwoImage from '@assets/images/placeholders/placeholder2.png';
+import PlaceholderThreeImage from '@assets/images/placeholders/placeholder3.png';
+
+const PLACEHOLDERS = [PlaceholderOneImage, PlaceholderTwoImage, PlaceholderThreeImage];
+
+const getRandomImagePlaceholder = (): StaticImageData =>
```

Comment:
RC: Let's make it more generic and make placeholders images array as a prop got getRandomImagePlaceholder. Now you are using this only with blog, but this method will be used also in another cases like team section so there will be different placeholders images

Also name of this method could be just getRandomImage.